### ISSUE TITLE
Switzerland hospitalisation data not getting updated

### ISSUE THREAD

**Issue by sbfnk** [covid19-forecast-hub-europe/covid19-forecast-hub-europe#1359]
It currently seems to rely on a date-fixed URL
https://github.com/epiforecasts/covid19-forecast-hub-europe/blob/92b1872a702a8b8698535d51154778af1b2d1a67/code/auto_download/hospitalisations/get-non-eu.R#L23

This code has previously been kindly suggested by a participant which seems to get around this issue:
```r
# Load libraries
library(lubridate)
library(httr)
library(jsonlite)

# Retrieve hospitalization data from Swiss FOPH
foph <- GET("https://www.covid19.admin.ch/api/data/context")
foph <- fromJSON(rawToChar(foph$content))

hosp <- read.csv(foph$sources$individual$csv$daily$hosp)
hosp$datum <- ymd(hosp$datum)
hosp <- subset(hosp, geoRegion == "CH", c("datum", "entries"))
names(hosp) <- c("date", "value")
```

**Comment by kathsherratt** [comment]
I'm very sorry, thank you for flagging this! I'll fix now.

**Comment by sbfnk** [comment]
Fixed via #1360
